AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOrchestrator.swift

```diff
@@ -70,6 +70,13 @@ final class AWSInitialSyncOrchestrator: InitialSyncOrchestrator {
         self.initialSyncOrchestratorTopic = PassthroughSubject<InitialSyncOperationEvent, DataStoreError>()
     }
 
+    deinit {
+        initialSyncOperationSinks.forEach { _, initialSyncOperationSink in
+            initialSyncOperationSink.cancel()
+        }
+        syncOperationQueue.cancelAllOperations()
```

Comment:
As above, let's make sure this actually gets called. If any operations on the queue are holding strong references to the orchestrator, it would result in a retain cycle that would mean `deinit` never gets invoked.